blinky/mix.exs

```diff
@@ -44,7 +44,7 @@ defmodule Blinky.MixProject do
       {:shoehorn, "~> 0.2"},
       {:nerves_runtime, "~> 0.4"},
       {:nerves_leds, "~> 0.8.0"},
-      {:nerves_init_gadget, "~> 0.3.0"}
```

Comment:
I'm partial to `{:nerves_init_gadget, "~> 0.4}` to save some time in the future.